osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/server/Iec60870ASduHandler.java

```diff
@@ -0,0 +1,38 @@
+package org.opensmartgridplatform.simulator.protocol.iec60870.server;
+
+import java.io.IOException;
+
+import javax.annotation.PostConstruct;
+
+import org.openmuc.j60870.ASdu;
+import org.openmuc.j60870.Connection;
+import org.openmuc.j60870.TypeId;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class Iec60870ASduHandler {
+
+    private final Logger logger;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1312)

Reply:
In this particular situation the Logger is set via the constructor by it's implementation classes...